workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts

```diff
@@ -62,14 +61,24 @@ describe('datasets create integration test', () => {
       expect(response.data).toBeDefined();
       expect(response.data.id).toBeDefined();
       expect(response.data.permissions).toBeDefined();
-      expect(response.data.permissions.length).toBe(2);
-      expect(response.data.permissions[1].accessLevel).toEqual('read-only');
-      expect(response.data.permissions[1].identityType).toEqual('USER');
+      expect(response.data.permissions.length).toBe(1);
       expect(response.data.permissions[0].accessLevel).toEqual('read-write');
       expect(response.data.permissions[0].identityType).toEqual('GROUP');
       expect(response.data.permissions[0].identity).toEqual(groupId);
```

Comment:
Since we're only expecting one object in the permissions array we could do
```
expect(response.data.permissions).toMatchObject<PermissionInterfaceName>({<permission data>});
```

Reply:
fixed.